rest-api-spec/src/main/resources/rest-api-spec/api/create.json

```diff
@@ -0,0 +1,75 @@
+{
+  "create": {
+    "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/master/docs-index_.html",
+    "methods": ["PUT"],
```

Comment:
we also except POST here in the java code. Not sure what the convention is though - do we want to be "correct" or reflect what he java does?